bcda/client/bluebutton.go

```diff
@@ -103,10 +103,16 @@ func (bbc *BlueButtonClient) GetPatient(patientID, jobID, cmsID string) (string,
 	return bbc.getData(blueButtonBasePath+"/Patient/", params, jobID, cmsID)
 }
 
-func (bbc *BlueButtonClient) GetPatientByHICNHash(hashedHICN string) (string, error) {
+func (bbc *BlueButtonClient) GetPatientByIdentifierHash(hashedIdentifier, patientIdMode string) (string, error) {
 	params := GetDefaultParams()
+
+	identifier := "hicnHash"
+	if patientIdMode == "MBI_MODE" {
+		identifier = "mbiHash"
```

Comment:
This should change to `mbi-hash` according to: https://github.com/CMSgov/beneficiary-fhir-data/blob/0877232340983fda8dbcd202012e991b8b531ac5/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java#L213

Reply:
also does this same stuff apply to the system mode we have in the models.go here https://github.com/CMSgov/bcda-app/blob/03491102295a8c6b250028a7b94e057268e24d66/bcda/models/models.go#L502

Reply:
> @tbellj
> Was this comment addressed? https://github.com/CMSgov/bcda-app/pull/432/files#r370441006
> I'm worried about the unit test coverage for both modes. Need to support both modes in the unit testing (now that we have test data for both) by setting the mode within the test.

nope, u right never addressed mostly because not sure what is needed honestly, we can talk tomorrow tho

Reply:
> bcda-app/bcda/models/models.go

Yup, need to use `hicn-hash` and `mbi-hash` according to the BFD code here: https://github.com/CMSgov/beneficiary-fhir-data/blob/0877232340983fda8dbcd202012e991b8b531ac5/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java